doc/Development.md

```diff
@@ -0,0 +1,30 @@
+# Development
+
+## Installation
+
+After cloning the repository into a directory called `RecordFlux`, `pip` can be used to install the project in editable mode:
+
+```Console
+pip install --user -e "RecordFlux[devel]"
+```
+
+This will also install all development dependencies, except for GNAT, which must be installed separately.
+
+On some Linux distributions (e.g., Debian, Ubuntu) a kernel parameter needs to be set in order that all tests can be executed successfully. The following command allows all users on the system to send ICMP messages using a special ICMP socket type.
+
+```
+sysctl -w net.ipv4.ping_group_range="0 2147483647"
```

Comment:
Maybe we should add `sudo` to indicate that this is privileged?

Reply:
What about prefixing commands with `$` or `#` instead?

https://wiki.archlinux.org/index.php/Help:Style#Command_line_text

Reply:
I'm fine with that, too.

Reply:
Fixed in 2562906.